### PR TITLE
Fix Bug 1245567 - Remove Firefox for Android 2.3 Gingerbread from www.mozilla.org

### PR DESCRIPTION
**Pull request by kyoshino**
This is due this Friday, April 29.

### CODE REVIEW

**Comment by kyoshino** [comment]
This has to be merged today, or there will be 404 links tomorrow. @pmclanahan @jgmize r?

File: bedrock/firefox/firefox_details.py [hunk 1]
```diff
@@ -310,8 +310,14 @@ class FirefoxAndroid(_ProductDetails):
     def platforms(self, channel='release'):
         platforms = self.platform_labels.copy()
 
+        # Android Gingerbread (2.3) is no longer supported as of Firefox 48
+        if int(self.latest_version(channel).split('.', 1)[0]) >= 48:
```
Comment (pmclanahan) [inline bedrock/firefox/firefox_details.py]: Looks like you're doing this split twice. Would be better to set a variable like:

```python
major_version = int(self.latest_version(channel).split('.', 1)[0])
```

**Comment by pmclanahan** [comment]
Apart from a minor thing I commented on this looks good. r+wc.

File: bedrock/firefox/firefox_details.py (hunk 1 above)
Comment (kyoshino) [inline bedrock/firefox/firefox_details.py]: Ah thanks! Fixed.

**Pull request merged**